lib/taiko.js

```diff
@@ -1885,6 +1893,7 @@ function dropDown(attrValuePairs, ...args) {
             const options = setNavigationOptions({});
             await doActionAwaitingNavigation(options, async () => {
                 const result = await runtimeHandler.runtimeCallFunctionOn(selectBox, null, { nodeId: nodeId, arg: value });
+                if(typeof result.result.value != 'boolean') throw new Error(result.result.value);
```

Comment:
This comparison is wrong again refer 
https://www.npmjs.com/package/apperror to see if this can be handled as an error and not using string comparison